sources/packages/backend/apps/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -102,6 +108,9 @@ export class InstitutionProfileAPIInDTO extends InstitutionContactAPIInDTO {
   website: string;
   @IsNotEmpty()
   regulatingBody: string;
+  @ValidateIf((e) => e.regulatingBody === "other")
+  @IsNotEmpty()
```

Comment:
Same max length validation needs to added here as well.

Reply:
I apologize, I missed it.

Reply:
No problem.